AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -516,9 +527,12 @@ async function validateConnectionStringAsync(propertyName, connectionString, typ
                     detailsMarkdown = title + `\r\n\r\n` + `This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  Relevant documentation:`
                         + `\r\n\r\n`;
                     switch (type) {
-                        case ConnectionStringType.StorageAccount:
+                        case ConnectionStringType.BlobStorageAccount:
                             detailsMarkdown += `<a href= "https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal" target="_blank">Storage account network security</a>`;
                             break;
+                        case ConnectionStringType.QueueStorageAccount:
+                            detailsMarkdown += `<a href= "https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal" target="_blank">Storage account network security</a>`;

```

Comment:
Unnecessary duplicate code.  The detailsMarkdown is the same for StorageAccount, BlobStorageAccount and QueueStorageAccount.  Use the switch case syntax to have the same logic for all three types.  What about FileStorageAccount?  This is not accounted for here?